index.html

```diff
@@ -0,0 +1,220 @@
+          minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+          aliquip ea eiusmod
+        </p>
+        <button type="submit">
+          Get my CV <i class="fa fa-download" aria-hidden="true"></i>
+        </button>
+      </section>
+      <div class="Code-lang">
+        <h3>Languages</h3>
+        <ul>
+          <li>JavaScript(ES5/ES6)</li>
+          <li>Ruby</li>
+          <li>HTML</li>
+          <li>CSS</li>
+        </ul>
+      </div>
+      <div class="Code-lang2">
+        <h3>Languages</h3>
+        <ul>
+          <li>JavaScript(ES5/ES6)</li>
+          <li>Ruby</li>
+          <li>HTML</li>
+          <li>CSS</li>
+        </ul>
+      </div>
+      <div class="Code-lang3">
+        <h3>Languages</h3>
+        <ul>
+          <li>JavaScript(ES5/ES6)</li>
+          <li>Ruby</li>
+          <li>HTML</li>
+          <li>CSS</li>
+        </ul>
+      </div>
+      <section class="Info-form">
+        <img src="img/image_geometry_1.png" alt="" />
+        <div class="footh2">
+          <h2>Get started</h2>
+        </div>
+        <div class="footP">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna.
+        </div>
+        <form action="https://formspree.io/f/{form_id}" method="post">
+          <label class="name" for="name">Your name</label>
+          <input
+            class="form"
+            type="text"
+            name="name"
+            placeholder=""
+            maxlength="30"
+            required
+          />
+          <label class="email" for="email">Your email</label>
+          <input
+            class="form"
+            type="email"
+            name="email"
+            placeholder=""
+            maxlength="30"
+            required
+          />
+          <textarea
+            name="message"
+            class="textarea"
+            placeholder="Enter your message here..."
+            cols="30"
+            rows="10"
+            maxlength="500"
+            required
+          ></textarea>
+          <button type="submit">Start collaboration</button>
+        </form>
+      </section>
+      <section class="socials">
+        <p>Get my Resume</p>
+        <ul class="icons">
+          <a href="https://github.com/SolomonTheCoder"
+            ><i class="fa fa-github fa-2x" aria-hidden="true"></i
+          ></a>
+          <i class="fa fa-stack-overflow fa-2x" aria-hidden="true"></i>
+          <a href="https://www.linkedin.com/in/solomon-afutu-164b95255/"
+            ><i class="fa fa-linkedin fa-2x" aria-hidden="true"></i
+          ></a>
+          <a href="https://twitter.com/SolomonAfutu"
+            ><i class="fa fa-twitter fa-2x" aria-hidden="true"></i
+          ></a>
+          <i class="fa fa-angellist fa-2x" aria-hidden="true"></i>
+          <i class="fa fa-medium fa-2x" aria-hidden="true"></i>
+        </ul>
```

Comment:
- [ ] However much you have added the social icons, they can not be displayed in the browser. Therefore ensure that they are matching the Figma design as we intend. Also, we need to use [Semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) guidelines. Hence, ensure that you refactor your semantics following this ` ul > li > a` ☺️ .

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/222087340-173c0e8f-4ae9-45b2-9cb4-56b32f60ff35.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/222087258-347529b2-e135-4ed5-9b18-1eaf43bbebfc.png)